Source/Core/DolphinWX/NetPlay/NetWindow.h

```diff
@@ -105,7 +105,6 @@ class NetPlayDialog : public wxFrame, public NetPlayUI
 	void OnTraversalError(int error) override;
 
 	static NetPlayDialog*& GetInstance() { return npd; }
-	static NetPlayClient*& GetNetPlayClient() { return netplay_client; }
```

Comment:
this functions just returns a global variable. the best fix probably would be to make it not global, but in the interest of making the least breaking change, I just used the global and got rid of the weird get-a-global-pointer-by-reference function and references to it